### DISCUSSION TITLE
stoichiometric reaction and competitive binding

### DISCUSSION THREAD

**Discussion by estelley1** [Open-Systems-Pharmacology/Forum#1575] [Questions & Problems] [answered]
Hi, 
I would like to create particular binding/complex formation reactions in MoBi and have several questions about stoichiometriy coefficients and how it is taken into account into the equations.

I have an antibody Ab with 2 binding sites (a and b) on one arm and each site can bind to receptor R.
            2*kon                                     kon
Ab + R <====> AbRa or AbRb  + R <========>AbRaRb
                  koff                                          2*koff

For the first part:
Does it make sense to create 1 reaction as follow : Ab + R => AbRa + AbRb ?
 Stoichiometry coefficients of 2 for Ab and R and 1 for AbRa and AbRb 
And do we write 2*kon in the equation formula, or is it already considered by the stoichiometry coefficients to consider the competitive binding?

Or does it make more sense to create 2 reactions as follow : Ab + R => AbRa and Ab + R => AbRb ?
 Stoichiometry coefficients of 1 for Ab, R, AbRa and also 1 for Ab, R, AbRb 
And do we write 2*kon in the equation formula to consider the competitive binding?

For the second part:
We have 2 reactions: AbRa + R => AbRaRb and AbRb + R => AbRaRb
 Stoichiometry coefficients of 1 for Ab, R, AbRa and also 1 for Ab, R, AbRb 
 Stoichiometry coefficients of 1 for Ab, R, AbRa and also 1 for Ab, R, AbRb 
And do we write 2*koff in the equation formula to consider the competitive dissociation?

Hope it is clear.
Thanks for your help.

**Comment by Yuri05** [comment, marked as answer]
>Hope it is clear.

no :)

So are you trying to create 2 binding reactions like this?
```math
2Ab + R \underset{\text{koff}}{\overset{\text{kon}}{\rightleftharpoons}} AbRa
```
```math
2Ab + R \underset{\text{koff}}{\overset{\text{kon}}{\rightleftharpoons}} 2AbRb
```

If so, then define 2 separate reactions:
* Reaction 1 with stoich coeffs 2,1,1 and reaction rate `kon*[Ab]*[R] - koff*[AbRa]`
* Reaction 2 with stoich coeffs 2,1,2 and reaction rate `kon*[Ab]*[R] - koff*[AbRb]`

**Reply by estelley1** [reply]
Thanks @Yuri05 :)
